rotkehlchen/tests/fixtures/accounting.py

```diff
@@ -54,7 +54,7 @@ def fixture_data_dir(use_clean_caching_directory, tmpdir_factory) -> Path:
     for x in data_directory.iterdir():
         directory_with_db = (
             x.is_dir() and
-            (x / 'rotkehlchen.db').exists() or (x / 'rotkehlchen_transient.db').exists()
+            ((x / 'rotkehlchen.db').exists() or (x / 'rotkehlchen_transient.db').exists())
```

Comment:
This was a bug. We needed `is_dir and (filename1 or filename2)` with a parentheses.